assets/controller.yaml

```diff
@@ -106,7 +106,6 @@ spec:
         - name: csi-provisioner
           image: ${PROVISIONER_IMAGE}
           args:
-            - --provisioner=csi.ovirt.org
             - --csi-address=$(ADDRESS)
             - --feature-gates=Topology=true
```

Comment:
I just realized that topology is true, but the CSI driver doesn't support it. Can we remove it as part of this PR?

Reply:
removed